web-server-lib/web-server/http/request.rkt

```diff
@@ -651,8 +613,8 @@
         (define line
           (begin
             ;; Skip the CRLF and then read the boundary line.
-            (read-http-line/limited in)
-            (read-http-line/limited in)))
+            (read-http-line/limited in #:limit max-field-length)
+            (read-http-line/limited in #:limit max-field-length)))
```

Comment:
@Bogdanp I'm less sure what the right limit is here. Previously it was `(current-http-line-limit)`, so `(* 8 1024)`.

Reply:
The call on line 616 should only ever read 2 bytes and the one after it should read up to an entire boundary line.  RFC2046 specifies that boundaries should be no longer than 70 characters so the total length of a boundary line can be up to 76 bytes long (two leading dashes + boundary + two final dashes (if final boundary) + CRLF). 

Since we know what the boundary is, we could do:

```racket
(read-http-line/limited in 2)  ;; CRLF
(read-http-line/limited in (+ (bytes-length end-boundary) 2))  ;; max boundary line length (including all the dashes) + CRLF
```

Reply:
Does the `#:limit` to `read-http-line/limited` include the terminating CRLF? I think it does, in which case I think reading the CRLF should be `(read-http-line/limited in #:limit 0)`.

Reply:
Yes, I tried it at the repl:
```
> (read-http-line/limited #:limit 2 (open-input-bytes #"--\r\n"))
#"--"
> (read-http-line/limited #:limit 0 (open-input-bytes #"\r\nabcdefg"))
#""
```